timeseries/src/autogluon/timeseries/transforms/covariate_scaler.py

```diff
@@ -13,35 +13,28 @@
 logger = logging.getLogger(__name__)
 
 
-class CovariateScaler:
+@runtime_checkable
+class CovariateScaler(Protocol):
     """Apply scaling to covariates and static features.
 
     This can be helpful for deep learning models that assume that the inputs are normalized.
     """
 
     def __init__(
```

Comment:
Should `__init__` be a part of the protocol? If so, then why do `CovariateRegressor` / `TargetScaler` not define `__init__` as a part of the protocol?